spec/support/missing_factory_helper.rb

```diff
@@ -0,0 +1,25 @@
+module Spec
+  module Support
+    module MissingFactoryHelper
+      def build(factory, *args, &block)
+        registered_factory_symbols.include?(factory) ? super : class_from_symbol(factory).new(*args)
+      end
+
+      def create(factory, *args, &block)
+        registered_factory_symbols.include?(factory) ? super : class_from_symbol(factory).create!(*args)
+      end
+
+      private
+
+      def class_from_symbol(symbol)
+        symbol.to_s.classify.constantize
+      end
+
+      def registered_factory_symbols
+        @registered_factory_symbols ||= FactoryGirl.factories.collect { |i| i.name.to_sym }
```

Comment:
Since the .include? in this could be called over and over, might be better to put it in a Set.